src/platform/plugins/shared/discover/public/embeddable/get_legacy_log_stream_embeddable_factory.ts

```diff
@@ -53,7 +54,7 @@ export const getLegacyLogStreamEmbeddableFactory = (
           attributes: {
             ...toSavedSearchAttributes(savedSearch, searchSourceJSON),
             references,
-          },
+          } as unknown as SearchEmbeddableSerializedState['attributes'],
```

Comment:
I'm not sure if this is still necessary or leftover from the pre-rebase branch. It should be looked into.